src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -2816,17 +2729,16 @@ Real64 GLHEVert::calcPipeConvectionResistance(EnergyPlusData &state)
 Real64 GLHEVert::frictionFactor(Real64 const reynoldsNum)
 {
     // Calculates the friction factor in smooth tubes
-
     // Petukov, B.S. 1970. 'Heat transfer and friction in turbulent pipe flow with variable physical properties.'
     // In Advances in Heat Transfer, ed. T.F. Irvine and J.P. Hartnett, Vol. 6. New York Academic Press.
 
-    // limits picked be within about 1% of actual values
+    // limits picked to be within about 1% of actual values
     constexpr Real64 lower_limit = 1500;
     constexpr Real64 upper_limit = 5000;
 
     if (reynoldsNum < lower_limit) {
         return 64.0 / reynoldsNum; // pure laminar flow
-    } else if (lower_limit <= reynoldsNum && reynoldsNum < upper_limit) {
+    } else if (reynoldsNum < upper_limit) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:2829]:(style),[knownConditionTrueFalse],Condition 'lower_limit<=reynoldsNum' is always true